common/djangoapps/django_sudo/utils.py

```diff
@@ -0,0 +1,15 @@
+"""
+sudo.utils
+"""
+import django.contrib.sessions.middleware
+import sudo.middleware
+
+
+def sudo_middleware_process_request(request):
+    """
+    Initialize the session and is_sudo on request object.
+    """
+    session_middleware = django.contrib.sessions.middleware.SessionMiddleware()
+    session_middleware.process_request(request)
```

Comment:
Is it necessary to explicitly call this? `django.contrib.sessions.middleware.SessionMiddleware` is included in `MIDDLEWARE_CLASSES` in both Studio and LMS.